src/Components/NavBar/Menus/NavBarUserMenu.tsx

```diff
@@ -61,20 +61,20 @@ export const NavBarUserMenu: React.FC = () => {
             CMS
           </NavBarMenuItemLink>
 
-          <Separator as="hr" my={1} />
-        </>
-      )}
-
-      {isCollectorProfileEnabled ? (
-        <>
           <NavBarMenuItemLink
             aria-label="View your purchases"
             to="/settings/purchases"
             onClick={trackClick}
           >
-            <ReceiptIcon mr={1} aria-hidden="true" /> Order History
+            Order History
           </NavBarMenuItemLink>
 
+          <Separator as="hr" my={1} />
+        </>
+      )}
```

Comment:
**Question:** why are we hiding the order history under `isAdmin` Or `hasPartnerAccess`?